src/application/services/useAuthRequired.ts

```diff
@@ -0,0 +1,40 @@
+import { useAppState } from './useAppState.ts';
+import useAuth from './useAuth.ts';
+import { useRouter } from 'vue-router';
+
+/**
+ * Function that is used in App for checking user authorization
+ * Works only for routes with authRequired set to true in route.meta
+ * If user is not authorized will show auth popup
+ */
+export default function useAuthRequired(): void {
+  const { showGoogleAuthPopup } = useAuth();
+  const router = useRouter();
+  const { user } = useAppState();
+
+  /**
+   * Check if user is authorized
+   * If authorization is in process we treat user as unauthorized
+   * When oauth will work, it will be treated as he authorized manually
+   * @returns true if user is authorized, false otherwise
+   */
+  function isUserAuthorized(): boolean {
+    return (user.value !== null || user.value === undefined);
```

Comment:
`user.value === undefined` — it is not authorized

Reply:
fixed